src/components/list/Item.js

```diff
@@ -22,7 +22,7 @@ export default function Item({ data }) {
           )}
           {data.icon && (
             <FontAwesomeIcon
-              icon={socialIcon(data.icon)}
+              icon={socialIcon(data.icon).icon}
               className="h-12 w-12 flex-none rounded-full bg-gray-50 p-2"
               style={{ height: 50, width: 50 }}
```

Comment:
Thank you for the latest changes. 

I just retested it, and noticed on the profile page it doesn't have the colour, any reason it was not added to this social icon?

Reply:
I have made the changes and colors reflect on profile and platform page as well

Reply:
Thank you